yarpcproto/yarpc.proto

```diff
@@ -3,9 +3,190 @@ syntax = "proto3";
 package uber.yarpc;
 
 option go_package = "yarpcproto";
+option java_multiple_files = true;
+option java_package = "com.uber.yarpc";
 
 // Oneway is the return type to use for an rpc method if
 // the method should be generated as oneway.
 message Oneway {
   bool ack = 1;
 }
+
+// ErrorType represents the type of error for an RPC call.
+//
+// Sometimes multiple error codes may apply. Services should return
+// the most specific error code that applies. For example, prefer
+// `OUT_OF_RANGE` over `FAILED_PRECONDITION` if both codes apply.
+// Similarly prefer `NOT_FOUND` or `ALREADY_EXISTS` over `FAILED_PRECONDITION`.
+enum ErrorType {
+  // Not an error; returned on success
+  //
+  // HTTP Mapping: 200 OK
+  // Google API Mapping: Code.OK
```

Comment:
Its actually google in this case https://godoc.org/google.golang.org/genproto/googleapis/rpc/code